### ISSUE TITLE
abs -c errase entire home directory!

### ISSUE THREAD

**Issue by 322sirc** [xerolinux/xero_xfce_iso#1]
Hello, Xero and Keyaedisa !
Just want to report to be very careful with the code abs: yesterday i build my Xero_Xfce iso with command given in the instructions and my entire /home/user/ was wiped off! I was under impression that the tool will make an special directory to build the iso and another one to release the iso and I was giving the home directory to abs to build ( as usual known in the build iso scripts) 
The abs tool has a command to erase totally the given build directory before starting the work of building the iso  --therefore  my entire home was flashed!
Then I tried building an iso given the same directory where i cloned xero_xfce_iso....again the same mistake...no result of iso.
The third attempt giving a diff directory than both above did the iso

**Comment by TechXero** [comment]
Hello...

Sorry about that, but you did not follow the prompts correctly. I build it 100 times a day. You need to specify full path for output and build directories.

Eg : /home/username/XFCE-Out/ and /home/username/XFCE-Build/

When you do that correctly it will work fine. I didn't know that prompts were too hard to understand...I will update the guide taking this into consideration.

**Comment by 322sirc** [comment]
Wasn't really clear in instructions to use different directory for build and out and yes I was given full path /home/username/... 
Luckily I build the iso in my Xero-kde installed partition and i didn't have much in home on it! I recovered the home with a skel and some of my customization files...

**Comment by TechXero** [comment]
It was mentioned in the prompt but I guess was unclear... Will update

**Comment by TechXero** [comment]
> Wasn't really clear in instructions to use different directory for build and out and yes I was given full path /home/username/... Luckily I build the iso in my Xero-kde installed partition and i didn't have much in home on it! I recovered the home with a skel and some of my customization files...

I have currently put project on hold, pending its completion.. I did not know it was gonna get any interest.. Will publish it once it's complete whitch it isn't.. It's 80% there.. Release will be May 15th.. Sorry about that...
